contentScript.js

```diff
@@ -0,0 +1,149 @@
+(() => {
+
+	let ytLeftControls, ytPlayer;
+	let currentVid = null;  // id of the currently playing video
+	let lastVid = null;   // id of the last handled video
+	let BmsList = [];  // list of the video bookmarks
+    
+		/* Fetch all bookmarks for current video */
+		const fetchBookmarks = () => {
+			return new Promise(resolve => {
+				// get all bookmarks for current video and create bookmark elements
+				chrome.storage.sync.get([ currentVid ], (obj) => {
+					resolve(obj[currentVid] ? JSON.parse(obj[currentVid]) : [])
+				});
+			});
+		}
```

Comment:
you can refactor this to a one liner.
 but more importantly i would format this